src/components/Reservations/MyReservations.js

```diff
@@ -1,8 +1,45 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import style from './MyReservation.module.css';
+import { convertDateWithName } from '../../helpers/format/format';
+
+import getReservations from '../../redux/reservation/reservationThunks';
 
 function MyReservations() {
+  const dispatch = useDispatch();
+  const reservation = useSelector((state) => state.reservationReducer);
+  useEffect(() => {
+    dispatch(getReservations());
+  }, []);
+
   return (
-    <div>MyReservations</div>
+    <div>
+      <table className={style.table} id="table">
+        <thead>
+          <tr>
+            <th>#</th>
+            <th>Doctor Picture</th>
+            <th>Doctor Name</th>
+            <th>Doctor Special</th>
+            <th>Appointment Date</th>
+            <th>action</th>
+          </tr>
+        </thead>
+        <tbody>
+          {reservation && reservation.map((element, i) => (
+            <tr key={element.id}>
+              <td>{i + 1}</td>
+              <td><img src={element.imageUrl} alt="doctors" className={style.img} /></td>
+              <td>{element.doctor.name}</td>
+              <td>{element.doctor.specialization}</td>
```

Comment:
Great job you have done so far in the doctors appointment page, but the name object does not exist in the reducer. Seems like you made the change in your backend for a branch that is not existing in the general repo yet. 

<img width="627" alt="Screenshot 2022-06-09 at 14 34 44" src="https://user-images.githubusercontent.com/82473943/172860622-bedfae26-39ad-4541-b8e8-cb303daefb55.png">

Kindly add the name property as an issue in the backend and create a pull request so we can have that corrected in the endpoint